arches_lingo/src/arches_lingo/components/schemes/SchemeBoxes.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import SchemeBox from "@/arches_lingo/components/schemes/SchemeBox.vue";
+import { fetchSchemes } from "@/arches_lingo/api.ts";
+
+const schemes = await fetchSchemes();
+</script>
+
+<template>
+    <div>
+        <ul
+            style="
+                display: flex;
+                flex-wrap: wrap;
+                list-style-type: none;
+                padding: 0;
+            "
+        >
+            <!-- Create New Scheme -->
+            <li class="scheme-card">Placeholder for creating new scheme</li>
+            <!-- Existing Schemes -->
+            <li
+                v-for="scheme in schemes"
+                :key="scheme.resourceinstanceid"
+                class="scheme-card"
+            >
+                <SchemeBox :scheme="scheme" />
+            </li>
+        </ul>
+    </div>
+</template>
+
+<style scoped>
+.scheme-card {
```

Comment:
ideally most of this is moved to the `SchemeBox` level. Also I'm wary of using `px` for min-width